{
  "endpoint": "/api/authentication/login",
  "method": "POST",
  "request": {
    "headers": {
      "X-Correlation-ID": "b207b952-0783-435e-962f-37f8c5e8f678",
      "X-Test-Context": "{\"featureFile\":\"invalid-credentials.api.feature\",\"scenario\":\"api/features/identity/authentication/edge-cases/invalid-credentials.api.spec.ts\",\"currentStep\":\"Test Execution\",\"testIntent\":\"negative\",\"expectedOutcome\":\"2xx_success\",\"testSessionId\":\"test-1757884771602-nlh2ucrhx\",\"testCase\":\"should handle multiple failed login attempts gracefully\"}"
    },
    "body": {
      "email": "test@example.com",
      "password": "***MASKED***"
    }
  },
  "response": {
    "status": 500,
    "error": {
      "error": "error",
      "message": "Authorization failed",
      "timestamp": "2025-09-14T21:19:31.6594634+00:00"
    }
  },
  "correlationId": "b207b952-0783-435e-962f-37f8c5e8f678",
  "timestamp": {},
  "duration": 112
}